graylog2-web-interface/src/views/pages/SearchPage.tsx

```diff
@@ -56,14 +57,15 @@ const SearchPageTitle = ({ children }: { children: React.ReactNode }) => {
 };
 
 const SearchPage = ({
-  children,
+  children = null,
   isNew,
   view: viewPromise,
   loadNewView: _loadNewView = defaultLoadNewView,
   loadView: _loadView = defaultLoadView,
   executionState: initialExecutionState = SearchExecutionState.empty(),
-  searchResult,
+  searchResult = null,
```

Comment:
I received this lint error once I modified this component. Is there any other better option other than defining a default null prop value?

Reply:
While `null` is okay, the goto way to deal with this is assigning `undefined`. This seems to be redundant, but the linter rule is supposed to remind of props which are optional and the default argument is forgotten. The linter cannot decide for us if the intention is to make it `undefined` if the prop is not passed or some other value, so we want to be reminded that we should double-check what the default value should be. This was actually the same with `defaultProps` as well.

Reply:
Thanks for the feedback @dennisoelkers. Makes sense. @zeeklop has the same feedback for me that `undefined` is the preferred default, but he also though that null might work slightly better in this case. Let me know if you think we are good with `null` or if I should update those to `undefined`.

Reply:
I went ahead and moved those props over to use `undefined`.